packages/serverpod/lib/src/database/adapters/postgres/database_connection.dart

```diff
@@ -674,7 +671,7 @@ class DatabaseConnection {
     return resolvedListRelations;
   }
 
-  void _validateColumnsExists(List<Column> columns, Table table) {
+  void _validateColumnsExists(Set<Column> columns, Table table) {
     for (var column in columns) {
       if (!table.columns.any((c) => c.columnName == column.columnName)) {
```

Comment:
Double iterations like these always makes me a little vary. I would like `table.columns` to be a set as well. Then this n-square loop could be replaced with a linear one (assuming a HashSet). Actually `A.intersection(B) == B` can be answered in log(n) time depending on the set implementation.

For most practical purposes it probably don't matter as the sets are small, but perhaps something to consider for later.